network-api/networkapi/wagtailpages/templates/wagtailpages/pages/dear_internet_page.html

```diff
@@ -83,8 +83,8 @@ <h2 class="mb-0">{{page.letters_section_heading}}</h2>
         {% if page.cta %}
         <p>{{page.cta}}</p>
         {% endif %}
-        {% if page.cta_button_text and page.cta_button_link %}
-        <a class="tw-btn-secondary tw-heart-glyph my-3" href="{{page.cta_button_link}}" target="_blank">{{page.cta_button_text}}</a>
+        {% if page.cta_button_text %}
```

Comment:
Like mentioned, we dont want this to link anywhere externally but instead push the relative url `?form=donate` so we can render the fundraise up modal